lib/Zureg/Views.hs

```diff
@@ -130,6 +131,15 @@ ticket r@Registrant {..} = template
 
         registrantInfo r
 
+        case rInfo of
+            Nothing                -> mempty
+            Just RegisterInfo {..} -> case riRegisteredAt of 
+                Nothing            -> mempty
+                Just rRegisteredAt -> when(rRegisteredAt >= Time.UTCTime (Time.fromGregorian 2019 5 7) (15*60*60)) $
+                    H.p $ do 
+                      "Please note that we have ordered the T-Shirts" 
+                      " and cannot guarantee that you will receive one if you register at this time."
```

Comment:
This note is what the participant sees when he looks at his ticket, so we should use a different message, e.g.:

> Because you registered after the T-Shirt were ordered, you will not be able to pick one up on the first day.